tech.json

```diff
@@ -861,6 +861,45 @@
               ]
             }
           ]
+        },
+        {
+          "name": "canEnterRMode",
+          "requires": [ "XRayScope", "ReserveTank", "canUseEnemies" ],
+          "note": [
+            "Ability to enter R-mode.",
+            "Have some energy in reserves (set to Auto), take damage while going through a door, use X-Ray while entering the next room, and release X-Ray after reserves have finished filling."
+          ],
+          "extensionTechs": [
+            {
+              "name": "canEnterGMode",
+              "requires": [ "canEnterRMode" ],
+              "note": [
+                "Ability to enter G-mode starting with low reserve energy (the usual way of entering G-mode).",
```

Comment:
The end bit still seems like it shouldnt be here.
"Ability to enter G-mode with little energy in reserve."

Reply:
Agreed, fixed.